packages/@okta/vuepress-site/docs/guides/oie-embedded-widget-use-case-sign-in-soc-idp/main/index.md

```diff
@@ -26,9 +26,9 @@ Set up your app to support the sign-in flow with Facebook IdP use cases by using
 
 ## Configuration updates
 
-Before you build the Facebook IdP sign-in flow using the Widget, ensure that you've [set up your Okta org for a social IdP use case](/docs/guides/oie-embedded-common-org-setup/-/main/#set-up-your-okta-org-for-a-social-idp-use-case).
+Before building the Facebook IdP sign-in flow using the widget, ensure that you've [set up your Okta org for a social IdP use case](/docs/guides/oie-embedded-common-org-setup/-/main/#set-up-your-okta-org-for-a-social-idp-use-case).
```

Comment:
@susanharper-okta I had to make an additional fix to try and get this page back up to 80 after I changed the title back.

Reply:
LGTM